tests/python_inline_comments/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -5,7 +5,9 @@ message = (
     "hello {}"  # comment
 ).format("world")
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-message = "hello {}".format("world") # comment
+message = (
```

Comment:
The output here changed for the same reason there parens are printed [on this line in comments.py](https://github.com/prettier/plugin-python/blob/dab3d3f5fe0ca3f5a2fcf3bc895f1a4c2e0b775a/tests/python_comments/__snapshots__/jsfmt.spec.js.snap#L23), which started happening because of a similar change I made to wrap parens around `if` `test` nodes. See my first comment on #79.

I did some more digging. The issue for both cases is that prettier includes a `breakParent` for all non-block comments [here](https://github.com/prettier/prettier/blob/3fc4c527d35d58f1d1163c24a4f8a4d1803cea0a/src/main/comments.js#L442), which triggers the parens `ifBreak`s. I haven't been able to come up with a workaround.

This PR does fix a case where invalid syntax is printed, so I still think it is net positive.